modules/cudaarithm/src/reductions.cpp

```diff
@@ -151,7 +151,12 @@ void cv::cuda::meanStdDev(InputArray src, OutputArray dst, Stream& stream)
     sz.width  = gsrc.cols;
     sz.height = gsrc.rows;
 
+#if (CUDA_VERSION >= 12040)
+    size_t bufSize;  // detail: CUDA>=12 NPP nppi_statistics_functions.h
```

Comment:
The comment in unnecessary because:
1) the #define implies that the type of `bufSize` depends on the version of CUDA,
2) the function is below so its clear where `bufSize` is used and
3) you should be able to navigate to the relavent header **which may change** using any IDE.